actions/crib-deploy-environment/action.yml

```diff
@@ -141,7 +141,7 @@ runs:
 
     - uses: actions/setup-go@v5
       with:
-        go-version-file: "${{ github.workspace }}/crib/go.work"
+        go-version-file: "${{ github.workspace }}/crib/cli/go.mod"
```

Comment:
should we change `cache-dependency-path` as well? just to keep both in sync

Reply:
nevermind, it actually makes sense to keep a cache of all golang apps inside the repo 👍